libraries/Microsoft.Bot.StreamingExtensions/IContentStream.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+
+namespace Microsoft.Bot.StreamingExtensions
+{
+    /// <summary>
+    /// Implemented by stream attachments compatible with the Bot Framework Protocol 3 with Streaming Extensions.
+    /// </summary>
+    public interface IContentStream
+    {
```

Comment:
This seems like it should be deleted. It's only used by ContentStream, and not by other classes such as HttpContentStream.

Reply:
It's used for testing/mocking